src/network/bridge.rs

```diff
@@ -251,6 +277,33 @@ impl driver::NetworkDriver for Bridge<'_> {
 
         let mut error_list = NetavarkErrorList::new();
 
+        let ipam = get_ipam_addresses(self.info.per_network_opts, self.info.network)?;
+        let if_name = self.info.per_network_opts.interface_name.clone();
+
+        // If we are using DHCP, we need to at least call to the proxy so that
+        // the proxy's cache can get updated and the current lease can be released.
+        if ipam.dhcp_enabled {
+            let dev = netns_sock
+                .get_link(netlink::LinkID::Name(if_name))
+                .wrap(format!(
+                    "get bridge interface {}",
+                    &self.info.per_network_opts.interface_name
+                ))?;
+
+            let container_mac_address = get_mac_address(dev.nlas)?;
+            release_dhcp_lease(
+                &self
+                    .info
+                    .network
+                    .network_interface
+                    .clone()
+                    .unwrap_or_default(),
+                &self.info.per_network_opts.interface_name,
+                self.info.netns_path,
+                &container_mac_address,
+            )?
+        }
```

Comment:
this block should be moved to a function in core_utils.rs so both bridge and maclvna code can call it and not have to duplicate it.